spec/lib/mcb/commands/apiv1/courses/find_spec.rb

```diff
@@ -8,6 +8,8 @@
 
     site_status2 = create(:site_status)
     course2 = site_status2.course
+    create_subject = create(:subject)
```

Comment:
it feels like the apiv1 specs are a good barometer to illustrate the whole of the course/site/subject 

extract of the code
```ruby
# no lonnger makes sense  # The site_status factory is an easy way to create a course and it's site
    site_status1 = create(:site_status)
    course1 = site_status1.course

    site_status2 = create(:site_status)
    course2 = site_status2.course
    create_subject = create(:subject)
    course2.update(subjects: [create_subject]) # was there really a need for this but?
    subject2 = course2.subjects.first

```

Reply:
@defong course2's subject is used later on when testing that it gets rendered out.

@davidgisbey does it make sense to have both `create_subject` and `subject2` here? We could probably just use `subject` instead of `create_subject`.

Reply:
It might make sense to create the `course` first so that we focus on that. Again, I started the pattern of creating `site_status` first, mea culpa ... I'm not sure that was a good idea any more.

Reply:
@misaka sounds like a plan. I'll go through my tests and sort them (there are a lot 😬 😆)